content/rc/0004-rede.md

```diff
@@ -139,24 +139,24 @@ Visto que o interessante de uma máscara são o número de bits ligados (`n`), e
 ser representada ao lado de um endereço IP (`a.b.c.d`) da seguinte forma:
 
 ```
-a.b.c.d/n
+a.b.c.d /n
```

Comment:
```suggestion
a.b.c.d/n
```

I don't think I've ever seen CIDR notation with spaces :thinking: 
Neither `ip -c a` nor [Wikipedia](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) have spaces in the CIDR representation of IP addresses.